src/passes/SyntaxHighlighter.jl

```diff
@@ -150,5 +151,3 @@ add_pass!(PASS_HANDLER, "SyntaxHighlighter", SYNTAX_HIGHLIGHTER_SETTINGS, false)
 end
 
 end
-
```

Comment:
I wonder if these lines were chomped by my text editor - this was not on purpose.